Protocol.md

```diff
@@ -613,7 +613,7 @@ The general format for an extension frame is given below.
                           Depends on Extended Type...
 ```
 
-* __Frame Type__: (16) 0xFFFF for Extension Header.
+* __Frame Type__: (16) 0xFFFF
```

Comment:
Hmm, that seems to have regressed in merges along the way. I'm positive we fixed that a few weeks back. Fixing again ... thanks for flagging it.